lib/utils/ensData.ts

```diff
@@ -44,32 +42,40 @@ export interface ResolvedEnsData {
   owner?: string;
 }
 
-// ENS Domain Coins utils
-export enum DomainAddressesSupportedCryptocurrencies {
-  BTC = "BTC",
-  LTC = "LTC",
-  DOGE = "DOGE",
-  ETH = "ETH",
-  BNB = "BNB",
-}
+export const cryptocurrencies: { [k: string]: string } = {
+  BTC: "BTC",
+  LTC: "LTC",
+  DOGE: "DOGE",
+  ETH: "ETH",
+  BNB: "BNB",
+  ARB1: "ARB1",
+  OP: "OP",
+  MATIC: "MATIC",
+};
 
-export const cryptocurrencyToCoinType = {
-  [DomainAddressesSupportedCryptocurrencies.BTC]: "0",
-  [DomainAddressesSupportedCryptocurrencies.LTC]: "2",
-  [DomainAddressesSupportedCryptocurrencies.DOGE]: "3",
-  [DomainAddressesSupportedCryptocurrencies.ETH]: "60",
-  [DomainAddressesSupportedCryptocurrencies.BNB]: "714",
+export const cryptocurrenciesToCoinType: { [k: string]: string } = {
+  [cryptocurrencies.BTC]: "0",
+  [cryptocurrencies.LTC]: "2",
+  [cryptocurrencies.DOGE]: "3",
+  [cryptocurrencies.ETH]: "60",
+  [cryptocurrencies.BNB]: "714",
+  [cryptocurrencies.ARB1]: "2147525809",
+  [cryptocurrencies.OP]: "2147483658",
+  [cryptocurrencies.MATIC]: "2147483658",
 };
 
-export const cryptocurrenciesToSymbol = {
-  [DomainAddressesSupportedCryptocurrencies.BTC]: "₿",
-  [DomainAddressesSupportedCryptocurrencies.LTC]: "Ł",
-  [DomainAddressesSupportedCryptocurrencies.DOGE]: "Ð",
-  [DomainAddressesSupportedCryptocurrencies.ETH]: "Ξ",
-  [DomainAddressesSupportedCryptocurrencies.BNB]: "₿",
+export const cryptocurrenciesToSymbol: { [k: string]: string } = {
```

Comment:
This is not used anywhere right?

Reply:
right, but it was already in there, I'd put the new coins, just in case, @FrancoAguzzi we should remove this?

Reply:
There doesn't seem to be a reason to include a symbol component here. @FrancoAguzzi, could you help us understand if this is necessary?